manifests/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -47,7 +47,7 @@ data:
     cpu:
       targetAverageUtilization: 80
     deploymentLabels: {}
-    enableProtocolSniffingForInbound: false
+    enableProtocolSniffingForInbound: true
```

Comment:
is it intended that with this true, and ALL ports in the cluster named, we still have a http_inspector injected?

Reply:
Yes. It is intended